fix: Fix and simplify navigation instructions to Repository Dashboard (#1667)
codacy/docs: docs/repositories/repository-dashboard.md

The **Repository Dashboard** provides an overview of the repository code quality and items that require your attention.

> To access your Repository Dashboard, select a repository from the [Repositories list](../organizations/managing-repositories.md) or open any other page of a repository and select **Dashboard** on the left navigation sidebar.

Reviewer: I don't really know how to **open any other page of a repository**...

Would this work? The wording can be improved, though... 😄 
```suggestion
To access your Repository Dashboard, select a repository from the [Repositories list](../organizations/managing-repositories.md) or select **Dashboard** on the left navigation sidebar if you are navigating on the details of a repository.
```

Reviewer 2: Another option would be to simplify it and suggest just one way to do this:

```suggestion
To access your Repository Dashboard, select a repository from the [Repositories list](../organizations/managing-repositories.md) and select **Dashboard** on the left navigation sidebar.
```

Author: Indeed, this part sounded a bit strange and over complicated. I checked the oldest version of the docs to understand the origins of the current wording:

https://docs.codacy.com/v2.0.387-deprecated/user-guide/your-repositories/repository-dashboard-how-does-it-work/#how-can-i-access-the-repository-dashboard

I agree with @nicklem that it should be quite safe to simplify this and not mention the navigation "when the user is already in some other repository page". :raised_hands:
